src/test/python_tests/test_code_actions.py

```diff
@@ -31,6 +31,11 @@ def _expected_organize_imports_command():
         "command": "editor.action.organizeImports",
     }
 
+def _expected_fix_u_string():
+    return {
+        "title": f"{LINTER}: Run document formatting",
+        "command": "editor.action.formatDocument",
+    }
```

Comment:
This does not seem like the right expected object for this scenario. From the code in the server to handle redundant u string this should be using edits and not a command.

Reply:
fixed.